mcenter_cli/parallelm/mcenter_cli/command_agent.py

```diff
@@ -0,0 +1,22 @@
+from cliff.command import Command
+
+
+class AgentRegisterCommand(Command):
+    def get_parser(self, prog_name):
+        parser = super(AgentRegisterCommand, self).get_parser(prog_name)
+        parser.add_argument('agent', action='store', help='Agent address to register')
```

Comment:
Maybe provide an example on what is expected here -- for the agent address.

Reply:
Fixed